tests/integration/interop-debug-adapter-test.js

```diff
@@ -0,0 +1,222 @@
+import { setupTest } from 'ember-qunit';
+import { settled } from '@ember/test-helpers';
+import InteropDebugAdapter from 'ember-m3/adapters/interop-debug-adapter';
+import { module, test } from 'qunit';
+import sinon from 'sinon';
+import DefaultSchema from 'ember-m3/services/m3-schema';
+import DS from 'ember-data';
+
+const BOOK_MODEL_TYPE = 'com.example.bookstore.Book';
+
+const generateColumns = columnArray =>
+  columnArray.map(attribute => ({ name: attribute, desc: attribute }));
+
+module('integration/interop-debug-adapter', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.sinon = sinon.createSandbox();
+
+    this.Publisher = DS.Model.extend({
```

Comment:
For some reason, `DS` models aren't being recognized in the `debug-adapter` for this test, even though it works when I test it manually. Am I registering the model incorrectly?

Reply:
@SYU15 I'm not sure what you mean; CI is 🍏 so I presume your tests pass; can you add a failing test that illustrates the way in which `Publisher` is not being recognized?

Reply:
Sure will update the test to fail when I'm back from DTO

Reply:
thanks @SYU15!

Reply:
I've updated the test to have the expected value for `typesAdded`, which is failing. Lmk what you think.